salt/metalk8s/saltmaster/minion_configured.sls

```diff
@@ -0,0 +1,31 @@
+{% set saltminion_version = '2018.3.3'%}
```

Comment:
Don't we want all these versions to be centralized in some default file, at least per state? It would help a lot if one day we want to upgrade those versions.

Reply:
Yes, in a couple of places already. However, no clear way how to achieve that (for now).

Reply:
#640